.github/workflows/rollbackDB.yml

```diff
@@ -7,8 +7,24 @@ on:
         description: 'The Liquibase tag to roll back to'
         required: true
       environment:
-        description: 'The environment of the DB that should be rolled back'
+        type: choice
         required: true
+        description: 'The environment of the DB that should be rolled back'
+        options:
+        - ""
+        - demo
+        - dev
+        - dev2
+        - dev3
+        - dev4
+        - dev5
+        - dev6
+        - dev7
+        - pentest
+        - test
+        - training
+        - stg
+        - prod
```

Comment:
This can be reverted, I just wanted a dropdown because I was running the job so frequently during testing.